### ISSUE TITLE
Pagination Component - WCAG Updates

### ISSUE THREAD

**Issue by rosiegrant** [yext/answers-search-ui#368]
Should've caught this earlier, however our WCAG-ness of pagination is not quite up to snuff

Could we follow the guidelines here:
https://www.a11ymatters.com/pattern/pagination/

From my understanding:
1. Should be wrapped in a navigation element
2. Add aria labels ("Go to First Page", "Go to Previous Page", "Go to Next Page", "Go to Last Page")

I'm testing using the Wave extension, https://wave.webaim.org/extension/.

**Comment by rosiegrant** [comment]
(once the eng working on this has made this update, pls test with wave and make sure there are no errors on the pagination component)

**Comment by rosiegrant** [comment]
closing, in qa spreadsheet now
